src/components/IGVBrowser.js

```diff
@@ -30,9 +30,9 @@ export const IGVBrowser = ({ selectedFiles, refGenome, namespace }) => {
       return {
         type: fileTypeToTrack[_.last(filePath.split('.'))],
         name: filePath,
-        url: filePath,
+        url: `${filePath}?userProject=${namespace}`,
         // igv.js will automatically find the index file, but not for crams
-        indexURL: filePath.endsWith('cram') ? filePath + '.crai' : ''
+        indexURL: filePath.endsWith('cram') ? `${filePath}.crai?userProject=${namespace}` : ''
```

Comment:
My understanding of the bucket billing is that specifying a project will always charge that project, even if the requested bucket is not Requester Pays. Can we verify the behavior of the fetch that IGV is doing under the hood, and make sure this change won't unnecessarily charge the current project for all accesses?

Reply:
Shoot - you're right. My understanding was the exact opposite, but according to [this documentation](https://cloud.google.com/storage/docs/requester-pays) your understanding is correct:
`Important: Buckets that have Requester Pays disabled still accept requests that include a billing project, and charges are applied to the billing project supplied in the request. Consider any billing implications prior to including a billing project in all of your requests.`

I will remove this and any associated code from this PR, and re-think my approach.